mpo.rga.Dayon.yml

```
@@ -0,0 +1,61 @@
+app-id: mpo.rga.Dayon
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: /app/dayon_assistant.sh
```

```suggestion
command: dayon_assistant.sh
```

the `.sh` should be in `/app/bin`